frappe/model/db_query.py

```diff
@@ -272,7 +272,7 @@ def prepare_args(self):
 				fields.append(field)
 			elif "as" in field.lower().split(" "):
 				col, _, new = field.split()
-				fields.append(f"`{col}` as {new}")
+				fields.append(f"`{col}` as `{new}`")
```

Comment:
what if the `new` already have backticks? It fails


```
select `name` as ``u``
                        from `tabUser`


                         order by `tabUser`.`modified` DESC None

```

Reply:
this query is works automatically , so no one can modify it by his own way
unless if he only changed this file

Reply:
@sharifaw no. I meant what about existing code that already had backticks to avoid this, this change will add double backticks. Try this;

```
frappe.get_all("User", ["name as `n`"]`
```